mappings/net/minecraft/item/Item.mapping

```diff
@@ -117,15 +117,19 @@ CLASS net/minecraft/unmapped/C_vorddnax net/minecraft/item/Item
 	METHOD m_obcnhrty damage (Lnet/minecraft/unmapped/C_sbxfkpyv;)Z
 		COMMENT {@return whether this item can be damaged by the given {@link DamageSource source}}
 		ARG 1 source
-	METHOD m_pfwgoldc onStackClicked (Lnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_nhvqfffd;Lnet/minecraft/unmapped/C_qcuteihm;Lnet/minecraft/unmapped/C_jzrpycqo;)Z
-		ARG 1 stack
-		ARG 2 slot
+	METHOD m_pfwgoldc onThisClickedOnStack (Lnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_nhvqfffd;Lnet/minecraft/unmapped/C_qcuteihm;Lnet/minecraft/unmapped/C_jzrpycqo;)Z
+		COMMENT Called when an {@link ItemStack} of this item is placed on another {@link ItemStack} in a slot.
+		COMMENT {@return whether an action was performed.}
+		ARG 1 thisStack
+		ARG 2 otherSlot
 		ARG 3 clickType
 		ARG 4 player
-	METHOD m_pktzlvtx onClicked (Lnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_nhvqfffd;Lnet/minecraft/unmapped/C_qcuteihm;Lnet/minecraft/unmapped/C_jzrpycqo;Lnet/minecraft/unmapped/C_xkkpnyvk;)Z
-		ARG 1 stack
+	METHOD m_pktzlvtx onStackClickedOnThis (Lnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_nhvqfffd;Lnet/minecraft/unmapped/C_qcuteihm;Lnet/minecraft/unmapped/C_jzrpycqo;Lnet/minecraft/unmapped/C_xkkpnyvk;)Z
```

Comment:
I really don't like this name.

The name doesn't make you think that this also called with an empty cursor, while it will.